src/components/AppController/AppController.js

```diff
@@ -23,13 +23,13 @@ var App = React.createClass({
       <Grid fluid>
         <Row>
           <Col lg={4} md={4}>
-            <Selector label={"Model Selection"} onChange={this.updateSelection.bind(this, 'model_id')} items={this.getMetadataItems('model_id')}/>
+            <Selector label={"Model Selection"} onChange={this.updateSelection.bind(this, 'model_id')} items={this.getMetadataItems('model_id')} value={this.state.model_id ? this.state.model_id : null}/>
```

Comment:
`this.state.model_id ? this.state.model_id : null` is more compactly expressed as `this.state.model_id || null`

Deeper questions: Do you really need to convert a falsy value to `null`? And note that it would convert a model id of `0` to `null`, also, which might not be desirable. Depends on type and range of values for that variable.